stable/voyager/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: Voyager provides controller for Ingress and Certificates for Kubernetes developed by AppsCode.
 icon: https://cdn.appscode.com/images/icon/voyager.png
 name: voyager
-version: 0.1.0
+version: 0.2.0
```

Comment:
What if we version Chart same as the Voyager version? And update this with every release of Chart
?